spec/src/modules/catalog.js

```diff
@@ -568,30 +861,33 @@ describe('ConstructorIO - Catalog', () => {
           fetch: fetchSpy,
         });
 
-        catalog.addItemsBatch({ items, section: 'Products' }).then(done);
+        catalog.createOrReplaceVariations({ variations, section: 'Products' }).then(done);
       });
 
-      it('Should resolve when removing multiple items', (done) => {
+      it('Should resolve when removing multiple variations', (done) => {
         const { catalog } = new ConstructorIO({
           ...validOptions,
           fetch: fetchSpy,
         });
 
-        catalog.removeItemsBatch({ items, section: 'Products' }).then(done);
+        variations.push(...variationsToCleanup);
+        catalog.deleteVariations({ variations: variations.map((variation) => ({ id: variation.id })), section: 'Products' }).then(() => {
```

Comment:
Can we follow the same patterns like we did in `deleteItems` where we checked that the fetchSpy was called, etc?